squiggy/lib/previews.py

```diff
@@ -38,7 +38,7 @@
 
 def generate_previews(object_id, object_url, object_type='asset'):
     if not app.config['PREVIEWS_ENABLED']:
-        return
+        return True
```

Comment:
When preview-image feature is disabled, we do not want invoking code to report an error.